source/getting-started/robot-nervous-system/blackboard.md

```diff
@@ -154,26 +508,28 @@ while node.wait(Duration::from_millis(update_rate_handle.get())).is_ok() {
 ```{grid-item}
 ### {octicon}`code` Rust
 
-{octicon}`mark-github` [Minimal Creator/Writer](https://github.com/eclipse-iceoryx/iceoryx2/blob/main/examples/rust/blackboard/creator.rs)  
-{octicon}`mark-github` [Minimal Reader](https://github.com/eclipse-iceoryx/iceoryx2/blob/main/examples/rust/blackboard/opener.rs)
+{octicon}`mark-github` [Minimal Creator/Writer](https://github.com/eclipse-iceoryx/iceoryx2/blob/main/examples/rust/blackboard/creator.rs)
```

Comment:
@FerdinandSpitzschnueffler FYI removing the two spaces at the end of the line broke the formatting. Two spaces in markdown signify a new line. I guess this was auto-formatted.